lib/validation-error.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+function ValidationError (errors, options) {
```

Comment:
Good idea to have a custom object for errors, we use ES2015 so can you do it by extending the Error class ? http://stackoverflow.com/questions/31089801/extending-error-in-javascript-with-es6-syntax